### PR TITLE
Added tidk/plot

### LINKED ISSUE #4537: new module: tidk/plot
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by GallVp**
## PR checklist

Closes #4537 

- [x] Added tidk/plot with nf-tests.
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/tidk/plot/meta.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "tidk_plot"
+description: write your description here
```
Comment (leoisl) [inline modules/nf-core/tidk/plot/meta.yml:4]: Please add `tidk/plot` description

File: modules/nf-core/tidk/plot/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    name "Test Process TIDK_PLOT"
+    script "../main.nf"
+    process "TIDK_PLOT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "tidk"
+    tag "tidk/plot"
+
```
Comment (leoisl) [inline modules/nf-core/tidk/plot/tests/main.nf.test:11]:
```suggestion
tag "tidk/search"

```

Fixes linting error: `Tags do not adhere to guidelines. Tags missing in main.nf.test: tidk/search`

File: modules/nf-core/tidk/plot/tests/main.nf.test [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                    []
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert process.out.svg != null },
```
Comment (leoisl) [inline modules/nf-core/tidk/plot/tests/main.nf.test:66]: Recent version (from yesterday) of `nf-core` linting will no longer fail if an empty file md5sum is found in a stub test, so this line could potentially be replaced by `{ assert snapshot(process.out).match() }` if you want. But this is minor, if you prefer the stub test as is, let's keep it as is!

**Review by leoisl — CHANGES_REQUESTED** [review]
Just two things, 1 which is to add a description for this subtool and another fixing a linting error. And then one minor thing on the stub test. All the rest looks great, thanks a lot!

File: modules/nf-core/tidk/plot/meta.yml (hunk 1 above)
Comment (mahesh-panchal) [reply modules/nf-core/tidk/plot/meta.yml:4]: Thanks. Nice catch.

Comment (GallVp) [reply modules/nf-core/tidk/plot/meta.yml:4]: Thank you Team!

File: modules/nf-core/tidk/plot/tests/main.nf.test (hunk 3 above)
Comment (GallVp) [reply modules/nf-core/tidk/plot/tests/main.nf.test:66]: I have applied the change.

**Review by leoisl — APPROVED** [review]
LGTM! Thanks a lot for your work on all these `tidk` modules!

**Pull request merged**